### PR TITLE
Nah recommended

### LINKED ISSUE #1: Add recommended products to user profile
Currently, when `/profile` is requested from the API, the products that the user has recommended to other users is included in the response.

```json
"recommends": [
    {
        "product": {
            "id": 32,
            "name": "DB9"
        },
        "customer": {
            "id": 5,
            "user": {
                "first_name": "Joe",
                "last_name": "Shepherd",
                "email": "joe@joeshepherd.com"
            }
        }
    }
]
```

This response needs to be augmented to also include the products that were recommended by another user to this user. 

1. The property should be `recommended`
2. It should be an array of recommendations that were made to the current user, not by them

```json
"recommendations": [
    {
        "product": {
            "id": 50,
            "name": "Escalade EXT"
        },
        "recommender": {
            "id": 7,
            "user": {
                "first_name": "Brenda",
                "last_name": "Long",
                "email": "brenda@brendalong.com"
            }
        }
    }
]
```

### PR DESCRIPTION
**Pull request by nathanRomad**
Updated Profile.py (ViewSet) and Customer.py (Models) to provide Recommendations when GET Profile is run.
Recommendations now provides BOTH: the products the user recommends AND the products recommended to that user.

## Changes
- added property to Customer model for 'recommended'
- added setter to Customer model for 'recommended'
- added RecommendedSerializer to Profile.py
- added field, 'recommended' to ProfileSerializer
- added 'recommended' to Response in Profile.py

## Requests / Responses
**Request**
GET `/profile` Fetches a user profile

```
http://localhost:8000/profile
```

**Response**
HTTP/1.1 200 OK

```json
{
    "id": 4,
    "url": "http://localhost:8000/customers/4",
    "user": {
        "first_name": "Steve",
        "last_name": "Brownlee",
        "email": "steve@stevebrownlee.com"
    },
    "phone_number": "555-1212",
    "address": "100 Infinity Way",
    "payment_types": [],
    "recommends": [],
    "recommended": [
        {
            "product": {
                "id": 1,
                "name": "Optima"
            },
            "recommender": {
                "id": 5,
                "user": {
                    "first_name": "Joe",
                    "last_name": "Shepherd",
                    "email": "joe@joeshepherd.com"
                }
            }
        }
    ]
}
```

## Testing
- open Postman
- select Bangazon Python API
- select Profile
- select GET User Profile
- change Authorization under Headers to: 'Token 9ba45f09651c5b0c404f37a2d2572c026c146690'
- click 'send' in postman to initiate fetch call

Data should come back with recommendations.

## Related Issues
- Fixes #1

### CODE REVIEW

**Comment by nathanRomad** [comment]
I only seeded ONE recommendation.....
So you will only see the single incoming recommendation for the user token provided.

File: bangazonapi/models/customer.py
Comment (Scott47) [inline bangazonapi/models/customer.py]: Please delete this commented out code before merge

**Review by Scott47 — APPROVED** [review]
👍 Other than the commented out code in your Customer model, this looks good. Good to merge once you delete the commented out code.

**Comment by nathanRomad** [comment]
commented code has been removed sir.

**Pull request merged**
